IPython/utils/traitlets.py

```diff
@@ -1635,14 +1635,14 @@ def validate_elements(self, obj, value):
 class Dict(Instance):
     """An instance of a Python dict."""
 
-    def __init__(self, default_value=None, allow_none=True, **metadata):
+    def __init__(self, default_value={}, allow_none=True, **metadata):
```

Comment:
@jdfreder I think that should be fine, the default value should be an empty dict either way, so the only thing this should change is the behaviour if you explicitly call it as `Dict(None)`, and I think that's OK.